config/config.go

```diff
@@ -214,3 +213,12 @@ func EnableDebugMode() bool {
 func IsService() bool {
 	return GetConfig().GetBool(types.IS_SERVICE)
 }
+
+func CreateFileAppendMode(filename string) *os.File {
+
+	File, err := os.OpenFile(filename, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+	if err != nil {
+		log.Fatalf("Error in creating %v file", filename, err)
```

Comment:
I could be coming in without much context here, but `log.Fatal` causes the program to terminate abruptly. Is that an acceptable scenario here? Or should it follow a more standard error handling flow (by returning an error)? At minimum, `log.Panic` might be preferred because it may give calling routines a chance to recover and do a more orderly exit.